core/snippets/snippets_insert_raw_text.py

```diff
@@ -40,9 +47,17 @@ def insert_snippet_raw_text(body: str):
         left(stop.columns_left)
 
 
+def format_tabs(text: str) -> str:
+    """Possibly replaces tabs with spaces in the given text."""
+    spaces_per_tab: int = settings.get("user.snippet_raw_text_spaces_per_tab")
+    if spaces_per_tab < 0:
+        return text
+    return re.sub(r"\t", " " * spaces_per_tab, text)
```

Comment:
There's no need to use a regular expression here to replace a single character when `str.replace` will do.